sympy/stats/matrix_distributions.py

```diff
@@ -464,3 +464,109 @@ def MatrixNormal(symbol, location_matrix, scale_matrix_1, scale_matrix_2):
         scale_matrix_2 = ImmutableMatrix(scale_matrix_2)
     args = (location_matrix, scale_matrix_1, scale_matrix_2)
     return rv(symbol, MatrixNormalDistribution, args)
+
+#-------------------------------------------------------------------------------
+# Matrix Student's T distribution ---------------------------------------------------
+
+class MatrixStudentTDistribution(MatrixDistribution):
+
+    _argnames = ('nu', 'location_matrix', 'scale_matrix_1', 'scale_matrix_2')
+
+    @staticmethod
+    def check(nu, location_matrix, scale_matrix_1, scale_matrix_2):
+        if not isinstance(scale_matrix_1, MatrixSymbol):
+            _value_check(scale_matrix_1.is_positive_definite != False, "The shape "
+                                                              "matrix must be positive definite.")
+        if not isinstance(scale_matrix_2, MatrixSymbol):
+            _value_check(scale_matrix_2.is_positive_definite != False, "The shape "
+                                                              "matrix must be positive definite.")
+        _value_check(scale_matrix_1.is_square != False, "Scale matrix 1 should be "
+                                               "be square matrix")
+        _value_check(scale_matrix_2.is_square != False, "Scale matrix 2 should be "
+                                               "be square matrix")
+        n = location_matrix.shape[0]
+        p = location_matrix.shape[1]
+        _value_check(scale_matrix_1.shape[0] == p, "Scale matrix 1 should be"
+                                                   " of shape %s x %s" % (str(p), str(p)))
+        _value_check(scale_matrix_2.shape[0] == n, "Scale matrix 2 should be"
+                                                   " of shape %s x %s" % (str(n), str(n)))
+        _value_check(nu.is_positive, "Degrees of freedom must be positive")
```

Comment:
```suggestion
        _value_check(nu.is_positive != False, "Degrees of freedom must be positive")
```
Whenever using checks similar to is_something, test it by `!= False`, so that we are sure that if the required property is doubtful (i.e., returns `None`) we let it pass.

Reply:
Ok I'll be careful about this